File: CS50's_Introduction_to_CS/Week1/agree.c
//
// Created by saksham on 16-06-2021.
//

#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Getting char input from user.
    char c = get_char("Do you agree? ");

    // Analysing user input:
    if (c == 'y' || c == 'Y'){
        printf("Agreed!!\n");
    } else if (c == 'n' || c == 'N') {
        printf("Not agreed. :(\n");
    } else {
        printf("Invalid Input!!\n");
    }
}

File: CS50's_Introduction_to_CS/Week6/Week1/agree.py
# from cs50 import get_string

# Prompt user to agree
# s = get_string("Do you agree? ")
s = input("Do you agree? ")

# Check whether agreed
# if s == "Y" or s == "y":
if s.lower() in ["y", "yes"]:
    print("Agreed.")
# elif s == "N" or s == "n":
elif s.lower() in ["n", "no"]:
    print("Not agreed.")

